Remotion/Core/Core/Logging/Log4NetLog.cs

```diff
@@ -125,8 +125,9 @@ private LoggingEvent CreateLoggingEvent (Level level, int? eventID, object messa
       return loggingEvent;
     }
 
-    private void LogLoggingError (int eventID, Exception? exceptionObject, object message)
+    private void LogLoggingError (int eventID, Exception? exceptionObject, object? message)
     {
+      // TODO RM-7804: The nullability of message should be consolidated across logging APIs.
```

Comment:
```suggestion
```
RM-7804 can be deleted now or will be udpated to add dedicated null tests.

Reply:
Ah yes, I've removed it - but what dedicated null tests do you mean? Unit tests for the null behavior?

Reply:
correct. thing is, log4net does not have NRT and if we want to offiically support null as a value, we should test the api for this. and we already have a ticket lying around we can repurpose :D

Reply:
Makes sense